source/transfer_list.rst

```diff
@@ -907,23 +907,61 @@ This manifest contains the SPMC attribute node consumed by the SPMD
 
 This entry type holds an array of `entry_point_info_t` structures.
 `entry_point_info_t` is a TF-A-specific data structure [TF_BL31]_ used to
-represent the execution state of an image, that is, the state of general purpose
-registers, PC and SPSR.
+represent the execution state of an image; that is, the state of general purpose
+registers, PC, and SPSR.
 
 .. code:: c
 
     typedef struct entry_point_info {
         param_header_t h;
-        uintptr_t pc;
         uint32_t spsr;
+
     #ifdef __aarch64__
```

Comment:
Perhaps I am becoming confused by github's awful code-review system, but it looks like you still have the #ifdef in here. Instead, these two structures should be defined separately